Nos.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -194,8 +185,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/pointfreeco/swiftui-navigation",
       "state" : {
-        "revision" : "2ec6c3a15293efff6083966b38439a4004f25565",
-        "version" : "1.3.0"
+        "revision" : "2aa885e719087ee19df251c08a5980ad3e787f12",
+        "version" : "0.8.0"
```

Comment:
Oh wait -- we're on the latest pre-1.0 version. So this probably answers my previous question -- I'm assuming 1.0 was when the builds slowed down significantly.

Reply:
I didn't actually check, I just went back to what I know was working before, but I added a note to #1035 to see how far we can upgrade or if there's some way to upgrade and exclude the macros.